packages/atdgen-runtime/atdgen-runtime.2.0.0/opam

```diff
@@ -20,7 +20,7 @@ depends: [
   "ocaml" {>= "4.02.3"}
   "jbuilder" {>= "1.0+beta7"}
```

Comment:
```suggestion
  "jbuilder" {>= "1.0+beta9"}
```

jbuilder fails with the [Unknown field package](https://opam.ci.ocaml.org/github/ocaml/opam-repository/commit/97a35ce36bbc5a62adb912c1ef5feab46231a846/variant/opam-2.1,compilers,4.07,atdgen-runtime.2.0.0,lower-bounds) error